about/index.md

```diff
@@ -19,33 +23,40 @@ Structured Threat Information Expression (STIX™) is a structured language to r
 
 <br class="clear: both;" />
 
-### The STIX Community
+## The STIX Community
+
+STIX is a collaborative effort that is led by DHS but driven by an open community. 
 
-STIX is a collaborative effort that is led by DHS but driven by an open community. Community members make open-source contributions to STIX development and manage issue tracking for the STIX schemas, tools, specifications, and supporting information. Join the community:
+Individuals and organizations are invited to join the STIX community!
+
+Feature requests and bugfixes are managed using [open source repositories on Github](https://github.com/stixproject/schemas)
 
 - [Mailing List](https://stix.mitre.org/community/registration.html) — where community members discuss the latest drafts of the STIX schemas, specifications, utilities, technical documents, and other items integral to the ongoing development of STIX.
 - [Chat](https://gitter.im/STIXProject/schemas) - informal discussions and help
-- [Code Repositories](https://github.com/STIXProject/) — the central location for STIX Community members to make open-source contributions to STIX development and manage issue tracking for the STIX schemas, tools, specifications, and other supporting information and items.
+- [Code Repositories](https://github.com/STIXProject/) — Github organization for development of schemas, tools, specifications, and other supporting information and items.
```

Comment:
I know it's nitpicky but can we change "Github organization" back to "the central location"? It's just much more clear and I don't think most people know what "Github organization really means.